Auth0/Credentials.swift

```diff
@@ -53,8 +53,8 @@ public class Credentials: NSObject, JSONObjectPayload, NSSecureCoding {
     }
 
     convenience required public init(json: [String: Any]) {
-        var expiresIn: Date? = nil
```

Comment:
That was causing the linter warning, because it was initializing with `nil` an optional variable.